catalog/ui/search-ui-app/src/main/resources/features.xml

```diff
@@ -51,13 +50,14 @@
     </feature>
 
     <feature name="search-ui-app" version="${project.version}"
-             description="The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.\nThe left pane of the SearchUI contains basic fields to query the Catalog and other Sources. The right pane consists of a map.\nIt includes both standard (3d globe) and simple (text page) versions.">
+             description="The Search UI is an application that not only provides results in a html format but also provides a convenient, simple querying user interface.\nThe left pane of the SearchUI contains basic fields to query the Catalog and other Sources. The right pane consists of a map.\nIt includes both the Catalog UI (3d globe) and Simple Search UI (text page) versions.">
         <configfile finalname="${ddf.etc}/org.codice.ddf.ui.searchui.filter.RedirectServlet.config" override="false">mvn:ddf.ui/search-ui-app/${project.version}/config/redirect</configfile>
         <feature>catalog-app</feature>
         <feature>catalog-versioning-plugin</feature>
         <feature>catalog-core-validator</feature>
         <feature>catalog-ui</feature>
         <feature>simple-search-ui</feature>
+        <bundle>mvn:ddf.ui.search/search-redirect/${project.version}</bundle>
```

Comment:
Should this be pulled out into its own feature?

Reply:
With the deprecation of Standard Search, I think this is fine for now. 
I'm wondering if this will eventually go away or if we will continue to want to redirect for Intrigue.

Reply:
I'd think we want to start this with the catalog-ui feature since that is what we are redirecting to. But if its configurable, maybe not.

Reply:
Yes, it is configurable. I would think that a downstream project might want to redirect to another default search UI by having a custom configuration for `Catalog` -> `Search UI Redirect`. Wouldn't this meant that the redirect should be installed regardless of what search UI is being used (Standard, Intrigue, something else)?

Reply:
Wouldn't such a downstream distribution likely define their own features anyways?

Reply:
I think we should put it in a separate feature and add that feature here since it is not coupled to our UIs. If a downstream project had completely different UI, they could just reference the feature.